src/libraries/Common/src/System/Security/Cryptography/CryptographicAttributeObject.cs

```diff
@@ -7,7 +7,13 @@
 
 namespace System.Security.Cryptography
 {
-    public sealed class CryptographicAttributeObject
+#if BUILDING_PKCS
+    public
+#else
+    #pragma warning disable CA1510, CA1512
```

Comment:
Why do we need to suppress these if we are in System.Security.Cryptography?

Because S.S.C.Pkcs targets netstandard, the analyzer that tells you to use the throw helpers does not kick in. However when this is pulled in to S.S.Cryptography, the analyzer lights up because it is unaware the source file is being pulled in to a non-net project.